test/integration/localnet/ledger-api/paid-traffic-cost.test.ts

```
@@ -73,23 +73,50 @@ async function resolveLedgerUserId(client: ReturnType<typeof getClient>, validat
+      return { row: parsed.data, parseFailures, rawSubmissionIds };
     }
   }
-  return undefined;
+  return { row: undefined, parseFailures, rawSubmissionIds };
+}
+
+function extractRawSubmissionId(row: unknown): string | undefined {
+  if (!row || typeof row !== 'object') {
+    return undefined;
+  }
+  const completionResponse = (row as { completionResponse?: unknown }).completionResponse;
```

ESLint

[error] Use object destructuring.

(prefer-destructuring)